lib/request.js

```diff
@@ -165,6 +165,16 @@ Object.defineProperties(Request.prototype, {
       return this[kRouteContext].config.url
     }
   },
+  routeOptions: {
+    get () {
+      const context = this[kRouteContext]
+      const serverLimit = context.server.initialConfig.bodyLimit
+      const routeLimit = context._parserOptions.limit
+      return {
+        bodyLimit: ((routeLimit || serverLimit))
+      }
```

Comment:
Wasn't there discussion of this containing more than just `bodyLimit`?

Reply:
no could you please suggest to me

Reply:
https://github.com/fastify/fastify/issues/4375#issuecomment-1298091027

Reply:
> Some of the field can be calculated (e.g. bodyLimit) and removed (e.g. config, hooks, etc.)

should I remove only config and hooks or anything else from the route options

Reply:
`schema`, `config` which is duplicated.
The property that are `function` because it do not make sense to reading it and do something on it.